build-logic/src/main/kotlin/org.jabref.gradle.base.dependency-rules.gradle.kts

```diff
@@ -4,9 +4,12 @@ plugins {
     id("org.gradlex.java-module-dependencies") // only for mappings at the moment
 }
 
-// TODO remove to translate 'requires' from 'module-info.java' to Gradle dependencies
-//      and remove 'dependencies {}' block from build.gradle files
-javaModuleDependencies { analyseOnly = true }
+javaModuleDependencies {
+    // TODO remove to translate 'requires' from 'module-info.java' to Gradle dependencies
+    //      and remove 'dependencies {}' block from build.gradle files
+    analyseOnly = true
+    moduleNameToGA.put("jul.to.slf4j", "org.slf4j:jul-to-slf4j")
```

Comment:
Added this to address the warning  about `jul.to.slf4j` you currently see when running a build.